### DISCUSSION TITLE
Changing maximum margin values

### DISCUSSION THREAD

**Discussion by iTheMaverick** [twbs/bootstrap#39409] [Q&A]
Is there anyway to change the margin value past 5 using a custom .scss file?

**Comment by julien-deramond** [comment]
Hey @iTheMaverick 
Do you want to create a `.m-6` class for example?
If this is the case, please check our [Utility API documentation](https://getbootstrap.com/docs/5.3/utilities/api/#modify-utilities). You could either modify some utilities, or add a new key/value for the `$spacers`, etc.
